### DISCUSSION TITLE
custom_parameters.yaml for a solis 3kw hybrid inverter (RAI-3K-48ES-5G or similar) including holding registers

### DISCUSSION THREAD

**Discussion by kenchy** [StephanJoubert/home_assistant_solarman#412] [Show and tell]
The solis hybrid settings didn't quite work for me, I was getting 'lookup' in 'operating status' and 'storage control mode'

I've made a few guesses on what the values should be.

Also I wanted some of the holding registers containing the charging schedule (for overnight use etc.) so I've added them (once I worked out they were frame type 0x03.

Maybe this will save someone some time.

[custom_parameters.yaml.txt](https://github.com/StephanJoubert/home_assistant_solarman/files/12701583/custom_parameters.yaml.txt)

**Comment by Ian-Johnston** [comment]
Did you get your RAI-3K-48ES-5G working in HA?
Can you share your files?

I'm not getting any data via my S2-WL-ST stick........first issue is my serial number is a mix of letters and numbers.
What stick are you using?

Ian.

**Comment by kenchy** [comment]
it works fine. I think mine is an s3-wifi-st by the look of it.
